src/nac/forms.py

```diff
@@ -19,6 +19,20 @@ class Meta:
         fields = UserChangeForm.Meta.fields
 
 
+class DeviceSearchForm(forms.Form):
```

Comment:
the search function does not work on my site. Could you check this again.

Reply:
I just checked: The ssearch function is working on the current commit in "main" branch. Something in this branch seems to break it.